pyhelper_utils/shell.py

```diff
@@ -70,3 +73,46 @@ def run_command(
         return False, out_decoded, err_decoded
 
     return True, out_decoded, err_decoded
+
+
+def run_ssh_commands(
+    host: Host,
+    commands: list,
+    get_pty: bool = False,
+    check_rc: bool = True,
+    timeout: int = TIMEOUT_30MIN,
+    tcp_timeout: Optional[float] = None,
+) -> list:
+    """
+    Run commands on remote host via SSH
+
+    Args:
+        host (Host): rrmngmnt host to execute the commands from.
+        commands (list): List of multiple command lists [[cmd1, cmd2, cmd3]] or a list with a single command [cmd]
+            Examples:
+                 single command: shlex.split("sudo reboot"),
+                 multiple commands: [shlex.split("sleep 5"), shlex.split("date")]
+
+        get_pty (bool): get_pty parameter for remote session (equivalent to -t argument for ssh)
+        check_rc (bool): if True checks command return code and raises if rc != 0
+        timeout (int): ssh exec timeout
+        tcp_timeout (float): an optional timeout (in seconds) for the TCP connect
+
+    Returns:
+        list: List of commands output.
+
+    Raise:
+        CommandExecFailed: If command failed to execute.
+    """
+    results = []
+    commands = commands if isinstance(commands[0], list) else [commands]
```

Comment:
Same here, I don't understand why this is needed.

Reply:
This had to be undone, because of mypy failing ```pyhelper_utils/shell.py:108: error: Name "commands" already defined on line 80  [no-redef]
Found 1 error in 1 file (checked 6 source files)
```